### ISSUE TITLE
Do you get listenable output?

### ISSUE THREAD

**Issue by mazzzystar** [jtcramer/wavegan#1]
Hi I run your code and the output seems to be `silence`, do you succeed to get the same results as the author's demo ?

If yes how can this happen ? Thanks~

**Comment by vinodkkurmi** [comment]
@fancoo  @jtcramer  Hi ,have you got listenable output? 
I got output that seems to a noise only.
Thanks

**Comment by fancoo** [comment]
@jtcramer Try with more training time ~

**Comment by mazzzystar** [comment]
@vinodkkurmi Try with more training time ^_^

**Comment by vinodkkurmi** [comment]
@fancoo Thanks.  But I tried with 80k epoch. but loss goes saturated. Could you tell which dataset you have tried. I am using URMP dataset. 

These are the loss curve 
![loss](https://user-images.githubusercontent.com/20721737/43686797-151f3930-98e9-11e8-8b8f-e0c2321a944e.png)

**Comment by mazzzystar** [comment]
@vinodkkurmi 
I use the dataset the paper author offered at [wavegan](https://github.com/chrisdonahue/wavegan).

`sc09`: https://drive.google.com/open?id=1qRdAWmjfWwfWIu-Qk7u9KQKGINC52ZwB


Actually I also implemented it by myself. The repo is:
https://github.com/fancoo/WaveGAN-pytorch

Hope it helps you ^_^
